src/schema/contentful/phoenix.js

```diff
@@ -429,12 +475,24 @@ const resolvers = {
   PersonBlock: {
     photo: linkResolver,
     alternatePhoto: linkResolver,
+    showcaseTitle: person => person.name,
+    showcaseDescription: person =>
+      person.type.includes('member board')
+        ? person.description
+        : person.jobTitle,
+
+    showcaseImage: (person, _, context, info) =>
+      linkResolver(person, _, context, info, 'alternatePhoto'),
```

Comment:
Oof this is tricky because we use the actual `photo` for advisory board members.
Let's change this to:
```suggestion
      const fieldName = person.type === 'advisory board member' ? 'photo' : 'alternatePhoto';
      linkResolver(person, _, context, info, fieldName),
```